src/main/java/world/bentobox/bentobox/schems/Clipboard.java

```diff
@@ -413,15 +448,35 @@ private boolean copyBlock(Block block, Location copyOrigin, boolean copyAir, Col
             return true;
         }
 
+        // Block state
+        BlockState bs = block.getState();
+
         // Set block data
-        s.set("bd", block.getBlockData().getAsString());
+        if (bs.getData() instanceof Attachable) {
```

Comment:
getData() returns MaterialData, which is deprecated, but there is no other way to determine Attachable, so it has to be used.